composer.json

```diff
@@ -44,10 +44,7 @@
     "require": {
         "php": "^7.3",
         "nesbot/carbon": "^2.16",
-        "ext-json": "*",
```

Comment:
What is the reason for removing `ext-json` from requirements?

Reply:
No particular reason apart from personal preference. It's include required as part of `guzzlehttp/guzzle` so would be checked anyway.